domain/model/service/modelservice_test.go

```diff
@@ -262,8 +285,18 @@ func (d *dummyControllerModelState) GetModelState(_ context.Context, modelUUID c
 }
 
 type dummyModelState struct {
-	models map[coremodel.UUID]model.ReadOnlyModelCreationArgs
-	setID  coremodel.UUID
+	models      map[coremodel.UUID]model.ReadOnlyModelCreationArgs
+	setID       coremodel.UUID
+	constraints constraints.Value
+}
+
+func (d *dummyModelState) GetModelConstraints(context.Context) (constraints.Value, error) {
```

Comment:
Can we start the process of replacing this with gomock?

Reply:
Yes please. This domain is a pain to work with when having to rebase with renamings.